docs/twin/dependencies.md

```diff
@@ -1,25 +1,25 @@
-# OPC Device Management dependencies
+# OPC Twin dependencies
 
-The OPC UA Device Management services depend on several other services.  This includes
+The OPC Twin Microservices depend on many other Microservices. This includes
```

Comment:
Leave original line (just change UA DM to Twin) as the dependencies are not Microsservices and there arent that "many" :-)